forest/daemon/src/daemon.rs

```diff
@@ -310,7 +298,7 @@ pub(super) async fn start(config: Config, detached: bool) -> Db {
     if config.client.enable_rpc {
         let keystore_rpc = Arc::clone(&keystore);
         let rpc_listen = std::net::TcpListener::bind(config.client.rpc_address)
-            .unwrap_or_else(|_| cli_error_and_die("could not bind to {rpc_address}", 1));
+            .context("could not bind to rpc address")?;
```

Comment:
This is slightly different from the previous error message. Is the address included in the error message from `bind`? If not then maybe use `format!`.

Reply:
Fixed